src/Snppts/Snippets/Covid19TrackerUI.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+
+namespace Snppts.Snippets
+{
+    public class Covid19TrackerUI : IAmASnippet
+    {
+        public string Slug => "covid19-tracker-ui";
+
+        public string Title => "Covid19 Tracker App";
+
+        public string Description => "Covid19.Forms is a tracker app to keep updated with Covid19";
+
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("AnthonyLatty/Covid19.Forms");
+
+        public Uri ExternalUri => null;
+
+        public bool ContainsAndroidSample => true;
+
+        public bool ContainsiOSSample => true;
+
+        public bool ContainsUWPSample => false;
+
+        public IAmAnAuthor AuthorInfo => new MarcAnthonyLatty();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/AnthonyLatty/Covid19.Forms/blob/master/Artwork/artwork_droid_3.jpg"),
```

Comment:
You'll want to use a URL like this `https://raw.githubusercontent.com/AnthonyLatty/Covid19.Forms/master/Artwork/artwork_droid_3.jpg` notice the "raw" in the first part. Else the image get redirected to a page on GitHub and they won't show

Reply:
Thanks!